scripts/build

```diff
@@ -4,7 +4,12 @@ set -e
 source $(dirname $0)/lib/debug_functions
 source $(dirname $0)/lib/version
 
-cd $(dirname $0)/..
-mkdir -p bin
-echo Building submariner-engine version $VERSION
-CGO_ENABLED=0 go build -ldflags "-X main.VERSION=$VERSION" -o bin/submariner-engine main.go
+cd $(dirname $0)/
+
+./build-engine
+./build-routeagent
+
+if [[ $5 = operator ]]; then
```

Comment:
This needs to be handled differently now, we don’t have the arguments at this point.

Reply:
right, there's my:

Error response from daemon: No such image: quay.io/submariner/submariner-operator:dev

thanks, I guess I can pass down the arguments to this script?

Reply:
ok did that for now: https://github.com/submariner-io/submariner/pull/173/files#diff-799c33a5bd89478f978c187c2d3f9b79R10